public/js/components/SourceTabs.css

```diff
@@ -74,6 +74,11 @@
   transition: transform 0.5s ease;
 }
 
+.collapse-button-left:hover,
+.collapse-button-right:hover {
+  cursor: pointer;
+}
+
```

Comment:
Hm, this had never occurred to me before. I think the better UX decision would be for these to have cursor icons, but I understand James' hesitation to be consistent. 

If we add this (which I think would be good), can we open a corresponding MC bug for someone to add the cursor pointers elsewhere in the tools?